pkg/cache/snapshot.go

```diff
@@ -31,16 +31,16 @@ func (c *Cache) Snapshot() Snapshot {
 	snap := Snapshot{
 		ClusterQueues: make(map[string]*ClusterQueue, len(c.clusterQueues)),
 	}
-	for _, capacity := range c.clusterQueues {
-		snap.ClusterQueues[capacity.Name] = capacity.snapshot()
+	for _, clusterQueue := range c.clusterQueues {
```

Comment:
cq?

A general rule is that the size of the name of a variable should be proportional to the number of times the variable is used.

Reply:
sure, I was more focused on editing anything that said `cap` or something, but sure

Reply:
done